samples/azure-quantum/chemistry/QuantumPhaseEstimation/Molecule.ipynb

```diff
@@ -0,0 +1,1775 @@
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CPU times: user 2.48 ms, sys: 400 µs, total: 2.88 ms\n",
+      "Wall time: 252 ms\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "-1.1238301470356422"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "%%time\n",
+    "GetEnergyQPE.simulate(\n",
+    "    JWEncodedData=encoded_data_h2, \n",
+    "    nBitsPrecision=7,\n",
+    "    trotterStepSize=0.4,\n",
+    "    trotterOrder=1\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'CNOT': 99584,\n",
+       " 'QubitClifford': 50106,\n",
+       " 'R': 43724,\n",
+       " 'Measure': 317,\n",
+       " 'T': 0,\n",
+       " 'Depth': 0,\n",
+       " 'Width': 5,\n",
+       " 'QubitCount': 5,\n",
+       " 'BorrowedWidth': 0}"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "resources_h2_qpe = GetEnergyQPE.estimate_resources(\n",
+    "    JWEncodedData=encoded_data_h2, \n",
+    "    nBitsPrecision=7,\n",
+    "    trotterStepSize=0.4,\n",
+    "    trotterOrder=1\n",
+    ")\n",
+    "resources_h2_qpe"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "-1.1166856359999997"
+      ]
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "GetEnergyVQE.simulate(\n",
+    "    JWEncodedData=encoded_data_h2, \n",
+    "    theta1=0.001,\n",
+    "    theta2=-0.001,\n",
+    "    theta3=0.001,\n",
+    "    nSamples=1\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'CNOT': 0,\n",
+       " 'QubitClifford': 28,\n",
+       " 'R': 0,\n",
+       " 'Measure': 70,\n",
+       " 'T': 0,\n",
+       " 'Depth': 0,\n",
+       " 'Width': 4,\n",
+       " 'QubitCount': 4,\n",
+       " 'BorrowedWidth': 0}"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "resources_h2_vqe = GetEnergyVQE.estimate_resources(\n",
+    "    JWEncodedData=encoded_data_h2, \n",
+    "    theta1=0.001,\n",
+    "    theta2=-0.001,\n",
+    "    theta3=0.001,\n",
+    "    nSamples=1\n",
+    ")\n",
+    "resources_h2_vqe"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Comparing resources\n",
+    "\n",
+    "The below bar chart shows the comparison between number of Clifford gates for the different algorithms on both Caffeine and H2."
```

Comment:
If I understand right, this is per job submission (that is, max depth)? Seems really low for VQE over all to use 28 operations, since at the least you need the overhead for binomial accuracy (1% → ~ 10k shots).

Reply:
The goal is not binomial accuracy, it's to see how many gates it requires per run. Adding more gates is difficult on near-term hardware but scaling the number of shots is trivial.

Reply:
Makes sense, but I think that may not have been clear from the notebook as written, such that clarifying that the resources count for VQE is measuring depth and not the total cost may be helpful here.

Reply:
Done, thanks!